### PR TITLE
feat: Progressive file download

### PR DESCRIPTION
**Pull request by KillianCourvoisier**
This PR change the way we downloading files and saving them.
Until then, we were intercepting the blob to of each file to pass it to the pilot. Now we can download file directly into the worker so we just need to give to it the download URL of each file instead of working with their blob.

We also save files one by one to appear faster on the user's cozy.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -428,21 +422,16 @@ class TemplateContentScript extends ContentScript {
...
           ...computedBill
         }
+        const fileurl = `${BASE_CLIENT_URL}${detailedFilepath}`
         detailedBill.filename = await getFileName(
           date,
           amount,
           currency,
-          detailed
+          detailed,
+          fileurl
```
Comment (doubleface) [inline src/index.js:431]: Why did you add the fileurl parameter here ?

Comment (KillianCourvoisier) [reply src/index.js:431]: I think it was in the action, i got no good reasons 👀 
Needs to be removed 👍
